doc/user/content/sql/create-sink/kafka.md

```diff
@@ -60,12 +60,16 @@ _item&lowbar;name_ | The name of the source, table or materialized view you want
 
 ### `CONNECTION` options
 
-Field                      | Value  | Description
----------------------------|--------|------------
-`TOPIC`                    | `text` | The name of the Kafka topic to write to.
-`COMPRESSION TYPE`         | `text` | Default: `none`. The type of compression to apply to messages before they are sent to Kafka: `none`, `gzip`, `snappy`, `lz4`, or `zstd`.
-`TRANSACTIONAL ID PREFIX`  | `text` | The prefix of the transactional ID to use when producing to the Kafka topic.<br>Default: `materialize-{REGION ID}-{CONNECTION ID}-{SINK ID}`
-`PROGRESS GROUP ID PREFIX` | `text` | The prefix of the consumer group ID to use when reading from the progress topic.<br>Default: `materialize-{REGION ID}-{CONNECTION ID}-{SINK ID}`
+Field                               | Value  | Description
+------------------------------------|--------|------------
+`TOPIC`                             | `text`              | The name of the Kafka topic to write to.
+`COMPRESSION TYPE`                  | `text`              | Default: `none`. The type of compression to apply to messages before they are sent to Kafka: `none`, `gzip`, `snappy`, `lz4`, or `zstd`.
+`TRANSACTIONAL ID PREFIX`           | `text`              | The prefix of the transactional ID to use when producing to the Kafka topic.<br>Default: `materialize-{REGION ID}-{CONNECTION ID}-{SINK ID}`
+`PROGRESS GROUP ID PREFIX`          | `text`              | The prefix of the consumer group ID to use when reading from the progress topic.<br>Default: `materialize-{REGION ID}-{CONNECTION ID}-{SINK ID}`
+`TOPIC REPLICATION FACTOR`          | `int`               | The replication factor to use when creating the Kafka topic (if applicable).<br>Default: Broker's default
+`TOPIC PARTITION COUNT`             | `int`               | The partition count to use when creating the Kafka topic (if applicable).<br>Default: Broker's default
+`PROGRESS TOPIC REPLICATION FACTOR` | `int`               | The partition count to use when creating the Kafka [progress topic](#exactly-once-processing) (if applicable).<br>Default: Broker's default
+`TOPIC CONFIG`                      | `map[text => text]` | Any topic-level configs to use when creating the Kafka topic (if applicable).<br>See [the kafka docs](https://kafka.apache.org/documentation/#topicconfigs) for available options.
```

Comment:
```suggestion
`TOPIC REPLICATION FACTOR`          | `int`               | The replication factor to use when creating the Kafka topic (if the Kafka topic does not already exist).<br>Default: Broker's default
`TOPIC PARTITION COUNT`             | `int`               | The partition count to use when creating the Kafka topic (if the Kafka topic does not already exist).<br>Default: Broker's default
`PROGRESS TOPIC REPLICATION FACTOR` | `int`               | The partition count to use when creating the Kafka [progress topic](#exactly-once-processing) (if the Kafka topic does not already exist).<br>Default: Broker's default
`TOPIC CONFIG`                      | `map[text => text]` | Default: empty<br>Any topic-level configs to use when creating the Kafka topic (if the Kafka topic does not already exist).<br>See the [Kafka documentation](https://kafka.apache.org/documentation/#topicconfigs) for available configs.
```

Took me a second to realize what you meant by "if applicable". I think it's worth being a bit more verbose but fully explicit. Also I think worth saying the default for the map is empty.